lib/utils.js

```diff
@@ -95,3 +95,62 @@ exports.findAvailablePort = function(app){
   });
 
 }
+
+exports.autoStoreData = function (req, res, next) {
+
+  // Middleware - store any data sent in session
+
+  if (!req.session.data){
+    req.session.data = {};
+  }
+
+  for (var i in req.body){
+    if (i.indexOf("_") != 0){
```

Comment:
indexOf is built-in and can only return an integer, so you dont need to check for strict equals